src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -64,6 +67,13 @@ public class SnowflakeSinkConnector extends SinkConnector {
   // Using setupComplete to synchronize
   private boolean setupComplete;
 
+  // The id of this connector instance. Should only be reset on start
+  private String kcInstanceId;
+
+  // If this connector is configured to use streaming snowpipe ingestion. Potentially may be better
+  // as a enum if we decide to add additional ingestion methods
```

Comment:
nit: do we need this comment? `Potentially may be better as a enum if we decide to add additional ingestion methods`

Reply:
nope not necessary, removed